cli/command/stack/swarm/deploy_composefile.go

```diff
@@ -120,29 +131,32 @@ func propertyWarnings(properties map[string]string) string {
 	return strings.Join(msgs, "\n\n")
 }
 
-func getConfigDetails(composefile string, stdin io.Reader) (composetypes.ConfigDetails, error) {
+func getConfigDetails(composefiles []string, stdin io.Reader) (composetypes.ConfigDetails, error) {
 	var details composetypes.ConfigDetails
 
-	if composefile == "-" {
+	if len(composefiles) == 0 {
+		return details, errors.New("no composefile(s)")
+	}
+
+	if composefiles[0] == "-" {
```

Comment:
should we error here `if len(composefiles) > 1` ?

Reply:
ohh, right probably. How do we handle that on `docker-compose` ?

Reply:
https://github.com/docker/compose/blob/master/compose/config/config.py#L263-L269

Looks like if it's not `[]string{"-"}` then it tries it as a file now, and would fail later on with a file not found.